docs/source/starter/style_guide.rst

```diff
@@ -173,55 +151,57 @@ In practice, this code looks like:
 
 Forward vs training_step
 ========================
-
-We recommend using forward for inference/predictions and keeping ``training_step`` independent.
+We recommend using forward for inference/predictions and keeping training_step independent
```

Comment:
I prefer when training_step stands out. Might be worth using LightningModule.training_step method hyperlink.